docker-hub/api/dvp.yaml

```diff
@@ -336,6 +439,48 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ResponseData'
+  /repos/pulls:
+    get:
+      tags: [namespaces]
+      summary: Get pull data for multiple repos
+      description: Gets pull for the given repos
+      operationId: getManyReposPulls
+      parameters:
+        - in: query
+          name: repos
+          schema:
+            type: array
+            items:
+              type: string
+          required: true
+          description: Repositories to fetch data for (maximum of 50 repositories per request).
+        - in: query
+          name: timespan
+          schema:
+            $ref: '#/components/schemas/TimespanType'
+          required: false
+          description: Timespan type for fetching data
+        - in: query
+          name: period
+          schema:
+            $ref: '#/components/schemas/PeriodType'
+          required: false
+          description: Relative period of the period to fetch data
+        - in: query
+          name: group
+          schema:
+            $ref: '#/components/schemas/GroupType'
+          required: false
+          description: Field to group the data by
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReposPullData'
+        '404':
```

Comment:
I think this is an error in the original docs. AFAIK we never return a 404, but an empty list